autogen.sh

```diff
@@ -122,7 +122,9 @@ echo "##########################################"
 echo
 
 # ucx
+if [ -f src/mpid/ch4/netmod/ucx/ucx/configure.ac ] ; then
```

Comment:
I believe bourne shell requires this format:

`if test -f ...`

Reply:
I just went with what I saw on line 101. I can change it if you want.

Reply:
Yikes!  It's all over the place.  Let's go ahead for now, but we'll need to do a cleanup (in a separate PR).